packages/flutter/test/material/theme_test.dart

```diff
@@ -254,13 +255,13 @@ void main() {
     );
     await tester.pump(const Duration(milliseconds: 100)); // Halfway through the theme transition
 
-    glyphText = tester.renderObject(find.byType(RichText));
+    glyphText = tester.renderObject(find.byType(RichText, skipOffstage: false));
```

Comment:
The vast majority of tests won't need to do this (you can see that I only needed to update a few tests). Only those tests that test something with `ExcludeSemantics` or `Offstage` _and_ want to probe into the offstage subtree need to opt into this behavior. Most tests are likely to be interested in onstage widgets only.